flight/targets/flyingf4/board-info/pios_board.h

```diff
@@ -223,7 +223,27 @@ extern uintptr_t pios_com_debug_id;
 //-------------------------
 #define PIOS_ADC_SUB_DRIVER_MAX_INSTANCES       3
 
-#define VREF_PLUS				3.0f
+// PIOS_ADC_PinGet(0) = PC_11
+// PIOS_ADC_PinGet(1) = PC_12
+//-------------------------
+#define PIOS_DMA_PIN_CONFIG                                                                   \
```

Comment:
Sorry, can you please either align all the backslashes using tabs, or just put them at the end of line?  It's a little ugly how they're on the right and don't line up.  After that, I'll be glad to merge.

Thanks for the additional F4 support!